.github/workflows/testcoverage.yml

```diff
@@ -0,0 +1,43 @@
+name: testcoverage
```

Comment:
this ain't a coverage test.

Reply:
This is true. Renamed it to a more appropriate name.